Fixed some typos and formatting. (#9)
danielehc/katacoda-scenarios: vault-consul-acl/intro.md

# Secure Consul Agent Communication with ACL

In this hands-on lab, you will deploy a secure Consul

> datacenter using Vault to generate and manage certificates.

Reviewer: ```suggestion
datacenter using Vault to generate and manage ACL tokens.
```

> The lab will guide you through the steps necessary to
> deploy Consul with TLS encryption enabled to secure access
> to the UI, API, CLI, services, and agents.
> 

Reviewer: ```suggestion
The lab will guide you through the steps necessary to deploy Consul
with ACL enabled to secure access to the cluster data and define agents' permissions.

<!-- Suggestion
I'm not sure "verify identity" is the correct phrasing. Maybe something like "deploy a Consul datacenter with ACLs enabled to secure access to the cluster data and agents". Cluster data meaning the catalog. Agents meaning accessing (managing) them with the UI/CLI/API.
-->

```

Reviewer 2: I'm not sure "verify identity" is the correct phrasing. Maybe something like "deploy a Consul datacenter with ACLs enabled to secure access to the cluster data and agents". Cluster data meaning the catalog. Agents meaning accessing (managing) them with the UI/CLI/API.

Specifically, you will:

> 

Reviewer: ```suggestion
- Start a Vault dev instance
- Start a Consul datacenter with ACL enabled
- Bootstrap ACLs in Consul
- Create a Consul policy for servers
- Enable Consul secrets engine in Vault
- Create a management token for Vault
- Create a Vault role to map Consul policy
- Create a Vault token associated with the role
- Verify the token was present in Consul and applied it to the agent
```

Reviewer 2: In the second point, `ACL` -> `ACLs`

> - Start a Vault dev instance
> - Create a policy in Vault to allow certificate generation

Reviewer: ```suggestion
```

> - Enable the PKI engine in Vault
> - Initialize the CA and generate an intermediate certificate
> - Generate certificates for your Consul servers
> - Use consul-template to retrieve certificates at runtime
> - Perform a certificate rotation

Reviewer: ```suggestion
```
